lib/rules/no-autofocus-attribute.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const AstNodeInfo = require('../helpers/ast-node-info');
+const Rule = require('./_base');
+
+const errorMessage =
+  'No autofocus attribute allowed, as it reduces accessibility by moving users to an element without warning and context';
+
+module.exports = class NoAutofocusAttribute extends Rule {
+  visitor() {
+    return {
+      ElementNode(node) {
+        const autofocusNode = AstNodeInfo.findAttribute(node, 'autofocus');
+        if (autofocusNode) {
+          if (this.mode === 'fix') {
```

Comment:
Unfortunately, it's not safe to autofix this rule by deleting the attribute, as that would cause behavior changes. Autofixes are not supposed to cause behavior changes. This fix would be more appropriate for an [auto-suggestion](https://eslint.org/docs/developer-guide/working-with-rules#providing-suggestions) which supported by ESLint but unfortunately ember-template-lint doesn't have such a feature.